localisation/00_wc_wonders.csv

```diff
@@ -98,5 +98,5 @@ wonder_shrine_storm_desc;The Shrine of the Storms is a monumental temple located
 wonder_shrine_storm_construction_desc;;;;;;;;;;;;;;x
 # Ulduar;;;;;;;;;;;;;;
 wonder_ulduar;Ulduar;;Ulduar;;;;;;;;;;;x
-wonder_ulduar_desc;Ulduar is a Titan city, largest of the facilities on Azeroth, and the prison of the Old God Yogg-Saron. In the northern regions of the Storm Peaks sits this magnificent city. Multiple wings make it up due to its sheer size. On the eastern side the Halls of Lightning are found, inside them is the Throne of Loken. On the opposite western side of the city exists the Halls of Stone which provide direct access to the Forge of Wills. A missing section, long transported to the sky by the titan-forged sorceress Helya, is the Halls of Valor home of Odyn. With Odyn, Prime Designate, departed Yogg-Saron drips his power through his shackles, influencing the titan-forged city little by little. Madness awaits those who succumb to his whispers. ;;Ulduar ist der gr��te von Titanen geschmiedete Komplex auf Azeroth. Er befindet sich in den Sturmspitzen und umfasst insbesondere die Hallen der Steine und die Hallen des Blitzes.;;;;;;;;;;;x
+wonder_ulduar_desc;Ulduar is a titanic city, the largest of the facilities on Azeroth, and the prison of the Old God Yogg-Saron. This magnificent city sits in the northern regions of the Storm Peaks. Multiple wings make up most of its size. On the eastern side, the Halls of Lightning are found, inside them lies the Throne of Loken. On the opposite, western, side of the city exists the Halls of Stone, which provide direct access to the Forge of Wills. A missing section, long ago transported to the sky by the titan-forged sorceress Helya, are the Halls of Valor, the home of Odyn. With Odyn, the Prime Designate, gone, Yogg-Saron drips his power through his shackles, influencing the titan-forged city little by little. Madness awaits those, who succumb to his whispers.;;Ulduar ist eine titanische Stadt, die gr��te der Einrichtungen auf Azeroth und das Gef�ngnis des alten Gottes Yogg-Saron. Diese prachtvolle Stadt liegt in den n�rdlichen Regionen der Sturmgipfel. Mehrere Fl�gel machen den gr��ten Teil der Stadt aus. Auf der �stlichen Seite befinden sich die Hallen des Blitzes, in deren Inneren der Thron von Loken steht. Auf der gegen�berliegenden, westlichen Seite der Stadt befinden sich die Hallen des Steins, die einen direkten Zugang zur Schmiede des Willens bieten. Ein fehlender Teil, der vor langer Zeit von der titanengeschmiedeten Zauberin Helya in den Himmel bef�rdert wurde, sind die Hallen der Tapferkeit, die Heimat von Odyn. Mit Odyn, dem obersten W�chter, aus dem Weg, l�sst Yogg-Saron seine Macht durch seine Fesseln sickern und beeinflusst nach und nach die titanengeschmiedete Stadt. Der Wahnsinn erwartet all jene, die seinen Einfl�sterungen erliegen.;;;;;;;;;;;x
```

Comment:
```suggestion
wonder_ulduar_desc;Ulduar is a Titan city, largest of the facilities on Azeroth, and the prison of the Old God Yogg-Saron. In the northern regions of the Storm Peaks sits this magnificent city. Due to the sheer size, the titans divided the complex into wings. On the eastern side, the Halls of Lightning exists, inside them the Throne of Loken. The western side contains the Halls of Stone, which provide direct access to the Forge of Wills. A missing section, long transported to the sky by the titan-forged sorceress Helya, is the Halls of Valor, home of Odyn. With the departure of Odyn, Yogg-Saron trickles his power through his shackles, influencing the titan-forged city little by little. Madness awaits those who succumb to his whispers.;;Ulduar ist eine titanische Stadt, die gr��te der Einrichtungen auf Azeroth und das Gef�ngnis des alten Gottes Yogg-Saron. Diese prachtvolle Stadt liegt in den n�rdlichen Regionen der Sturmgipfel. Mehrere Fl�gel machen den gr��ten Teil der Stadt aus. Auf der �stlichen Seite befinden sich die Hallen des Blitzes, in deren Inneren der Thron von Loken steht. Auf der gegen�berliegenden, westlichen Seite der Stadt befinden sich die Hallen des Steins, die einen direkten Zugang zur Schmiede des Willens bieten. Ein fehlender Teil, der vor langer Zeit von der titanengeschmiedeten Zauberin Helya in den Himmel bef�rdert wurde, sind die Hallen der Tapferkeit, die Heimat von Odyn. Mit Odyn, dem obersten W�chter, aus dem Weg, l�sst Yogg-Saron seine Macht durch seine Fesseln sickern und beeinflusst nach und nach die titanengeschmiedete Stadt. Der Wahnsinn erwartet all jene, die seinen Einfl�sterungen erliegen.;;;;;;;;;;;x
```
I stand by the original suggestion. With this iteration, I fixed up the grammar and changed a few words. Also to note that Titan City is how Ulduar is spoken of.